src/genie/libs/parser/nxos/tests/test_show_interface.py

```diff
@@ -1424,6 +1424,85 @@ class TestShowRunInterface(unittest.TestCase):
         },
     }
 
+    golden_output_5 = {'execute.return_value': '''
```

Comment:
We no longer use this form of unittests. You can see an example of how we now do them here: https://github.com/CiscoTestAutomation/genieparser/tree/master/src/genie/libs/parser/nxos/tests/ShowRunningConfigNvOverlay/cli